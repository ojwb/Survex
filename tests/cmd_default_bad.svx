; pos=fail warn=3 error=3
*fix 1 0 0 0
1 2 1 0 0
*default calibrate data
*default foo
*default un1ts
