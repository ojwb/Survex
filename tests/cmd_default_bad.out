./cmd_default_bad.svx:4:2: warning: *DEFAULT is deprecated - use *CALIBRATE/DATA/SD/UNITS with argument DEFAULT instead
 *default calibrate data
  ^~~~~~~
./cmd_default_bad.svx:4:20: error: End of line not blank
 *default calibrate data
                    ^~~~
./cmd_default_bad.svx:5:2: warning: *DEFAULT is deprecated - use *CALIBRATE/DATA/SD/UNITS with argument DEFAULT instead
 *default foo
  ^~~~~~~
./cmd_default_bad.svx:5:10: error: Unknown setting "foo"
 *default foo
          ^~~
./cmd_default_bad.svx:6:2: warning: *DEFAULT is deprecated - use *CALIBRATE/DATA/SD/UNITS with argument DEFAULT instead
 *default un1ts
  ^~~~~~~
./cmd_default_bad.svx:6:10: error: Unknown setting "un1ts"
 *default un1ts
          ^~~~~

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 2 survey stations, joined by 1 leg.
There are 0 loops.
Total length of survey legs =    1.00m (   1.00m adjusted)
Total plan length of survey legs =    1.00m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from 2 at 0.00m to 2 at 0.00m)
North-South range = 1.00m (from 2 at 1.00m to 1 at 0.00m)
East-West range = 0.00m (from 2 at 0.00m to 2 at 0.00m)
   2 1-nodes.
There were 3 warning(s) and 3 error(s) - no output files produced.
